tokio-quiche/src/quic/connection/mod.rs

```diff
@@ -120,6 +130,12 @@ impl AsSocketStats for QuicConnectionStats {
             bytes_recvd: self.stats.recv_bytes,
             bytes_lost: self.stats.lost_bytes,
             bytes_retrans: self.stats.stream_retrans_bytes,
+            bytes_unsent: 0,
```

Comment:
Is it correct to hard code this to 0?

Reply:
This was implemented like that in the legacy codebase. I should add a comment.
AFAIK we don't have this value in quiche, but we have it for TCP sockets, so it's worth adding to `SocketStats` for compatibility.

Reply:
I would add a comment to the struct or code for posterity.